src/commands/config/mod.rs

```diff
@@ -46,7 +45,7 @@ pub fn global_config(user: &GlobalUser, verify: bool) -> Result<(), failure::Err
 // validate_credentials() checks the /user/tokens/verify endpoint (for API token)
 // or /user endpoint (for global API key) to ensure provided credentials actually work.
 pub fn validate_credentials(user: &GlobalUser) -> Result<(), failure::Error> {
-    let client = http::cf_v4_api_client(user, HttpApiClientConfig::default())?;
+    let client = http::auth_client(user)?;
```

Comment:
the distinction for this client really is that it is optimized as a client for the cloudflare API, not that it includes auth info, so i'd be inclined to leave this name as it was.